plugins/Minecraft/index.js

```diff
@@ -0,0 +1,86 @@
+const fetch = require('node-fetch');
+
+export default function (bastion, opt = {}) {
+    // const cmd = bastion.command("minecraft")
+    // const help = `${cmd} (description)`
+    return [{
+            command: "mc_info",
+            helpOnEmpty: false,
+            // restrict: [bastion.channels.gaming],
+            restrictMessage: "",
+            options: "",
+            validate(context, msg) {
+                if (msg.includes(" ")) return 'Please just do the command'
+            },
+
+            async resolve(context, msg) {
+                const mcURL = "https://mcapi.us/server/status?ip=51.161.122.136&port=25595";
+
+                let response = await fetch(mcURL);
+                if (response.ok) {
```

Comment:
to prevent deep nesting, which can make code hard to follow you can early exit instead of wrap the entire block in an if statement

```js
// If something went wrong, exit early
if (!response.ok) return;
```